src/audio/copier/copier.c

```diff
@@ -772,22 +772,47 @@ static int set_attenuation(struct comp_dev *dev, uint32_t data_offset, const cha
 	return 0;
 }
 
+static int copier_set_configuration(struct processing_module *mod,
+				    uint32_t config_id,
+				    enum module_cfg_fragment_position pos,
+				    uint32_t data_offset_size,
+				    const uint8_t *fragment, size_t fragment_size,
+				    uint8_t *response,
+				    size_t response_size)
+{
+	struct module_data *md = &mod->priv;
+	struct comp_dev *dev = mod->dev;
+
+	comp_dbg(dev, "copier_set_config()");
+
+	dcache_invalidate_region((__sparse_force void __sparse_cache *)fragment, fragment_size);
```

Comment:
I'm wondering why some `.set_configuration()` methods invalidate cache and others don't? If it's needed, maybe it should be done centrally in `module_set_large_config()`? @ranj063